app.py

```diff
@@ -234,25 +254,29 @@ def parse_date(d):
 
 
 # Takes list of events and returns list of events occuring in specified date range
-def filter_events_by_date(events, start_date_str=datetime.datetime.now(datetime.timezone.utc), end_date_str=None):
+    def filter_events_by_date(events, start_date_str=datetime.datetime.now(datetime.timezone.utc), end_date_str=None):
+        
+        # number of days specified in config
+        days = config.get('past_events', 'past_days')
 
-    if start_date_str:
-        start_date = parse_date(start_date_str) - datetime.timedelta(days=1)
-    else:
-        start_date = None
-    end_date = parse_date(end_date_str) if end_date_str else None
-
-    if isinstance(start_date, str) or isinstance(end_date, str):
-        return '{}{}'.format(start_date, end_date).replace('None', '')
-
-    if start_date and end_date:
-        return [event for event in events if start_date <= parse(event['time']) <= end_date]
-    elif start_date:
-        return [event for event in events if parse(event['time']) >= start_date]
-    elif end_date:
-        return [event for event in events if parse(event['time']) <= end_date]
-    else:
-        return events
+        if start_date_str:
+            start_date = parse_date(start_date_str) - datetime.timedelta(days = 3)
```

Comment:
Is this still hard-coded to 3, or is this saying use the days variable and default to 3 if it's not set?

If it's setting it to 3 then I suspect it would be something like below. Yes/no?
`datetime.timedelta(days = days_in_the_past)
`
?

Reply:
@allella 
Yes, you are correct. I'm a little confused what happened with this commit as these were all (except for the naming days_in_the_past) were already corrected. I could've gotten mixed up as well, with where I was to push.

I will make another commit/pull request for you to review.